tests/manual/tableproperties.js

```diff
@@ -27,13 +27,12 @@ ClassicEditor
 		],
 		table: {
 			contentToolbar: [ 'tableColumn', 'tableRow', 'mergeTableCells', 'tableProperties', 'tableCellProperties' ],
-			tableToolbar: [ 'bold', 'italic' ]
-		}
+			tableToolbar: [ 'bold', 'italic' ],
+		},
```

Comment:
dangling comma

Reply:
Check your IDE setting guys :D

Reply:
Can we configure ESLint to disallow it?

Reply:
Would be awesome :D

Reply:
My OCD would love this :P

Reply:
[There you go.](https://github.com/ckeditor/ckeditor5/issues/6246)